extensions/kill-process/src/index.tsx

```diff
@@ -64,6 +65,20 @@ export default function ProcessList() {
     return process.path == null ? null : <CopyToClipboardAction title="Copy Path" content={process.path} />;
   };
 
+  const subtitleStr = (process: Process) => {
```

Comment:
Just a _minor_ nitpick, but prefer not to use abbreviations, even `subtitle` would be good enough, but if you want to be explicit, `subtitleString` is even better.

Thanks for making it better!

Reply:
Done 😃 .